encoding/preparation.md

```diff
@@ -250,7 +251,85 @@ such as Intel's [SVT-AV1][] will also not be included.
 
 ## Audio Codecs
 
-TODO: FLAC, AAC, maybe mention opus
+### Audio formats and how to handle them
+
+Depending on the source you'll be working with,
+you may encounter many different audio formats.
+
+On Blu-rays,
+you will most likely find audio encoded losslessly,
+in the form of either DTS-HD Master Audio, Dolby TrueHD, or PCM.
+DTS-HD MA and Dolby THD are proprietary codecs that use lossless compression,
+while PCM is simply raw, uncompressed PCM data.
+The usual way to handle these formats is to reencode them to other formats -
+either lossless or lossy, depending on your taste.  
+But first, you need to decode them.
+The recommended tool for that is FFmpeg.
+You can find Windows builds and Linux packages
+on [FFmpeg's official site][ffmpeg].
+
+When working with WEB and TV sources,
+you will most likely have only lossy audio available.
+The most common codecs here are AC-3, E-AC-3 and AAC.
+Lossily compressed files should generally not be reencoded -
+the proper way to handle them is to remux (i.e. copy) them to the final file.
+
+[ffmpeg]: https://www.ffmpeg.org/download.html
+
+### Which codecs to use?
```

Comment:
Section headers/titles not directly under another header should have two blank lines before them, separating them from the previous section.